### DISCUSSION TITLE
Add F2 and F3 live timing

### DISCUSSION THREAD

**Discussion by Electricman37** [f1multiviewer/issue-tracker#110] [Feature requests]
Formula 2 and 3 have live timing available only via website and you can't stop/delay it there so it's always ahead of broadcast. I think it would be really great to have it in the app so we can sync it with the stream.

**Comment by stefanverkerknl** [comment]
This would really be a great addition! I really miss this sort of information when watching F2 / F3 races.

Maybe if a driver map is / driver locations are available, that would be great too.

**Comment by f1multiviewer** [comment]
Just wanted to give an update on this:

- I've tested with a new endpoint, which seems to return F2/F3 timing in a very similar format to F1, and this works great
- Position (for the map), car telemetry (for the RPM/gear/speed/DRS etc.), mini sectors, tire/stint information and team colors are all missing, resulting in a somewhat poor experience (though this is similar to the official live timing from F2/F3's websites)
- Some work still needs to be done, but I expect this to ship pretty soon
